### DISCUSSION TITLE
Turn off the ripple effect?

### DISCUSSION THREAD

**Discussion by modo1302** [MelleD/lovelace-expander-card#195] [Q&A] [answered]
Hi! First of all I want to thank you for this beautiful fork and it's maintenance. Just switched from. The original integration, it's like night and day! Excellent work!

Question: is there any easy way to turn off the ripple effect visible on the expand button?

**Comment by MelleD** [comment, marked as answer]
Hey @modo1302 sorry for the late answer. Didn’t saw this post 😅.
Maybe you can remove it with card mod. Currently there is no setting for it:

see https://github.com/MelleD/lovelace-expander-card/blob/main/src/ExpanderCard.svelte#L237
